src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
I am not sure if we need to keep IsNetNative in this condition. currently in F5 scenario (which is UAP) IsNetNative returns true. maybe the naming is wrong? UAP is not net native really but uapaot is.